src/index.html

```diff
@@ -9,6 +9,68 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="container__logo">
+        <img 
+          src="images/logo.png" 
+          alt="moyo-logo" 
+          class="header__img"
+        >
+      </a>
+      <nav>
```

Comment:
add class .nav to create a BEM block

Reply:
Hello! this theme BEM will be in the future, I am not familiar with it yet